packages/x-date-pickers-pro/src/SingleInputDateRangeField/tests/SingleInputDateRangeField.keyboard.test.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
```

Comment:
The test files are usually named `{topic}.{componentName}.test.tsx`
It is the nomenclature on the data grid and on the new pickers test files.

I would name this one `selection.SingleInputDateRangeField.test.tsx`

Reply:
For now, all the `x-pickers-pro` are like that. I will rename all of them in a follow up PR

Reply:
@LukasTy is creating new test files using the correct nomenclature in #7135
Can be done in a follow up PR of course